tmc5160/func_test.go

```diff
@@ -0,0 +1,61 @@
+//go:build test
```

Comment:
I think due to this directive, the tests are in this file are not being run by the CI.

See https://github.com/tinygo-org/drivers/pull/725/checks#step:7:182

If you want to exclude these tests from being run by the GH action you should add it here https://github.com/tinygo-org/drivers/blob/dev/Makefile#L22

It would be better if the tests can run, however. :smile_cat:

Reply:
I removed the file as it was not relevant.